frontend/packages/dev-console/src/components/add/AddPageLayout.scss

```diff
@@ -39,8 +41,10 @@
         display: flex;
       }
       &__text {
-        margin-left: var(--pf-global--spacer--xs);
-        vertical-align: middle;
+        // Used font-size-base variable instead of --pf-global--FontSize--sm, which is getting calculated to 13px instead of 14px
+        font-size: $font-size-base;
```

Comment:
Hmm, but we should try to avoid this cross package dependency when possible. How does it looks with 14 px? Maybe we / UX is fine with this as well?